core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -388,130 +364,137 @@ var ExplorationEditorTranslationTab = function() {
+      progressBarLabelElement)).toBe('Exploration translation progress:');
+    expect(await action.getAttribute(
+      'Translation mode button',
       translationModeButton,
-      'Translation mode button taking to long to appear');
-    expect(await translationModeButton.getAttribute('class')).toMatch(
-      'oppia-active-mode');
-    expect(await voiceoverModeButton.getAttribute('class')).not.toMatch(
-      'oppia-active-mode');
+      'class')).toMatch('oppia-active-mode');
+    expect(await action.getAttribute(
+      'Voiceover mode button',
+      voiceoverModeButton,
+      'class')).not.toMatch('oppia-active-mode');
   };
 
   this.expectToBeInVoiceoverMode = async function() {
-    await waitFor.visibilityOf(
-      languageSelectorLabelElement,
-      'Language selector label element taking too long to appear');
-    expect(await languageSelectorLabelElement.getText()).toBe(
-      'Voiceovers for language:');
-    expect(await progressBarLabelElement.getText()).toBe(
-      'Exploration voiceover progress:');
-    await waitFor.visibilityOf(
+    expect(await action.getText(
+      'Language selector label element',
+      languageSelectorLabelElement)).toBe('Voiceovers for language:');
+    expect(await action.getText(
+      'Progress bar element',
+      progressBarLabelElement)).toBe('Exploration voiceover progress:');
+    expect(await action.getAttribute(
+      'Translation mode button',
       translationModeButton,
-      'Translation mode button taking to long to appear');
-    expect(await translationModeButton.getAttribute('class')).not.toMatch(
-      'oppia-active-mode');
-    expect(await voiceoverModeButton.getAttribute('class')).toMatch(
-      'oppia-active-mode');
+      'class')).not.toMatch('oppia-active-mode');
+    expect(await action.getAttribute(
+      'Voiceover mode button',
+      voiceoverModeButton,
+      'class')).toMatch('oppia-active-mode');
   };
 
   this.expectContentTabContentToMatch = async function(content) {
-    await waitFor.elementToBeClickable(
-      contentTabButton, 'Content Tab button is not clickable');
-    await contentTabButton.click();
-    expect(await contentTabText.getText()).toMatch(content);
+    await action.click('Content tab button', contentTabButton);
+    expect(await action.getText(
+      'Content tab button',
+      contentTabText)).toMatch(content);
   };
 
   this.expectFeedbackTabContentsToMatch = async function(contents) {
-    await waitFor.elementToBeClickable(
-      feedbackTabButton, 'Feedback Tab button is not clickable');
-    await feedbackTabButton.click();
+    await action.click('Feedback tab button', feedbackTabButton);
     expect(await feedbackList.count()).toEqual(contents.length);
     for (var index in contents) {
-      await translationFeedback(index).click();
-      expect(await translationFeedbackText(index).getText()).toMatch(
-        contents[index]);
+      await action.click(
+        'Translation feedback button',
```

Comment:
Can you include the index in this message so we know which button was clicked?

Reply:
ya sure